global.js

```diff
@@ -1,6 +1,3 @@
-import { ENABLE_DEV_MODE } from 'react-native-dotenv';
-import { useTheme } from './src/context/ThemeContext';
-
 if (typeof btoa === 'undefined') {
   global.btoa = function (str) {
```

Comment:
I assume this one can't be moved. Might be better to move it to the shim itself so we don't have 2 "global" related files